salt/personalised-covers/aws.sls

```diff
@@ -0,0 +1,10 @@
+aws-credentials:
+    file.managed:
+        - name: /home/{{ pillar.elife.deploy_user.username }}/.aws/credentials
+        - source: salt://personalised-covers/config/home-user-.aws-credentials
+        - user: {{ pillar.elife.deploy_user.username }}
```

Comment:
a different `file.managed` to the home directory of the aforementioned user, finding it with http://stackoverflow.com/questions/23144324/how-to-i-retrieve-the-home-directory-of-a-user